ckan/logic/action/get.py

```diff
@@ -1761,6 +1761,10 @@ def package_search(context: Context, data_dict: DataDict) -> ActionResult.Packag
         sysadmin will be returned all draft datasets. Optional, the default is
         ``False``.
     :type include_drafts: bool
+    :param include_deleted: if ``True``, deleted datasets will be included in the
+        results (site configuration "ckan.search.remove_deleted_packages" must
+        be set to False). Optional, the default is ``False``.
+    :type include_deleted: bool
```

Comment:
```suggestion
    :param include_deleted: if ``True``, deleted datasets will be included in the
        results (the configuration option :ref:`ckan.search.remove_deleted_packages` must
        be set to ``False``, otherwise this parameter is ignored). Optional, the default is ``False``.
    :type include_deleted: bool
```